src/OpenTelemetry.Api/Trace/TelemetrySpan.cs

```diff
@@ -60,6 +60,38 @@ public SpanContext Context
             }
         }
 
+        /// <summary>
+        /// Gets the parent span context.
+        /// </summary>
+        public SpanContext ParentSpanContext
```

Comment:
Should we let people do `ParentSpan.Context`? I think on Activity you have to do similar.

Reply:
yeah, you are right. But, I don't know the answer.
@cijothomas , what do you think? Should we expose the property or should we let the user use `ParentSpan.Context`?

Reply:
There are active spec discussion in this area. Lets come back to it once it settles. We can close this PR now and pick up later.